mobile/App.js

```diff
@@ -29,7 +30,7 @@ const App = () => {
             component={ScanView}
             options={{tabBarStyle: globalStyleSheet.tabBarHidden}}
           />
-          <Tab.Screen name="My ICard" component={RegistrationView} />
+          <Tab.Screen name="My ICard" component={MyICardView} />
```

Comment:
The idea is "My ICard" page would show RegistrationView if user is not logged in, otherwise it will show MyICardView. Is this possible?

Reply:
This is possible. Because I thought this is just a layout, I didn't connect the authentication and the My ICard pgae together yet.

Reply:
Ok! I'm going to merge this and we can do it on the current sprint